src/chains/xna.ts

```diff
@@ -0,0 +1,81 @@
+/* source: https://github.com/RavenProject/Ravencoin/blob/master/src/chainparams.cpp */
+
+import { Common, Network, TestNetwork } from '../types';
+
+const common: Common = {
+  name: 'Neurai',
+  unit: 'XNA',
+  decimalPlaces: 1e8,
+  // https://github.com/NeuraiProject/Neurai/blob/master/src/validation.cpp#L129
+  messagePrefix: 'Neurai Signed Message:\n',
+};
+
+const main: Network = {
+  ...common,
+  // https://github.com/NeuraiProject/Neurai/blob/master/src/chainparams.cpp#L179
+  hashGenesisBlock:
+    '00000044d33c0c0ba019be5c0249730424a69cb4c222153322f68c6104484806',
+  // https://github.com/NeuraiProject/Neurai/blob/master/src/chainparams.cpp#L172
+  port: 19000,
+  // https://github.com/NeuraiProject/Neurai/blob/master/src//chainparamsbase.cpp#L36
+  portRpc: 19001,
+  protocol: {
+    magic: 0x5255454e,
+  },
+  // https://github.com/NeuraiProject/Neurai/blob/master/src/chainparams.cpp#L182
+  seedsDns: [
+    'seed1.neurai.org',
+    'seed2.neurai.org',
+    'neurai-ipv4.neuraiexplorer.com',
+  ],
+  versions: {
+    // https://en.bitcoin.it/wiki/BIP_0032
+    bip32: {
+      private: 0x0488ade4,
+      public: 0x0488b21e,
+    },
+    // https://github.com/NeuraiProject/Neurai/blob/master/src/chainparams.cpp#L200
+    bip44: 0,
```

Comment:
Thanks for the submission. Do you have a bip44 coin type set up? You can see and submit here as well: https://github.com/satoshilabs/slips/blob/master/slip-0044.md

Reply:
Neurai uses the derivation m/44'/0'/0'/0'/0, so it is not possible to add it to this list.